curiouslearning/FeedTheMonsterJS
# fix: CR-227 changes for matchfirst level  ([]) brackets

# Changes
- src/components/prompt-text.ts
- build/feedTheMonster.js

# How to test
- Check for the match first levels in Bangla and some other lanuages.

Ref: [CR-227](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-227)


[CR-227]: https://curiouslearning.atlassian.net/browse/CR-227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
